src/share/poudriere/jail.sh

```diff
@@ -740,6 +741,7 @@ install_from_ftp() {
 				;;
 			allbsd) URL="https://pub.allbsd.org/FreeBSD-snapshots/${ARCH%%.*}-${ARCH##*.}/${V}-JPSNAP/ftp" ;;
 			ftp-archive) URL="http://ftp-archive.freebsd.org/pub/FreeBSD-Archive/old-releases/${ARCH%%.*}/${ARCH##*.}/${V}" ;;
+			freebsdci) URL="https://artifact.ci.freebsd.org/snapshot/${V}/latest_tested/${FULLARCH%%.*}/${FULLARCH##*.}" ;;
```

Comment:
Everywhere I see `FULLARCH` it appears that `ARCH` suffices fine. The line right above this has the same expansions as this line.

Reply:
No, ARCH is set to only the first componant if we give TARGET.TARGET_ARCH which we need to do as both are in the url in the artifact urls.
I should add that to the man page.

Reply:
I think that the allbsd case doesn't work anymore.

Reply:
Mhm, wait, I might have misread the code.

Reply:
Yep sorry, fixed